ux.symfony.com/templates/base.html.twig

```diff
@@ -28,14 +28,15 @@
         {% endblock %}
     </head>
     <body>
+        <a href="#main" class="skip-to-content">Skip to main content</a>
         {% block body %}
             <div class="App">
                 {% block header %}
                     {{ include('_header.html.twig') }}
                 {% endblock %}
 
                 {% block main %}
-                    <main>
+                    <main id="main">
```

Comment:
Playing around, it seems like on most (all?) pages, the `main` element is already at the top of the page. So clicking the link doesn't actually do anything. Are we sure this will be helpful?

Reply:
For me it is just a quick win, we'll have a lot of work to be fully accessible, but it goes in the good direction.
* it is not visible per default
* it is usefull for screen readers / keyboard-only (see https://www.w3.org/TR/WCAG20-TECHS/G1.html)

But we could give it less "contrast" and keep the same background if you think it's too much ? Or see a bit later to handle the accessible question in a more "site-wide" way ?

Reply:
Let's remove it for now then!

The other modifications will have a direct impact so i'll update the PR is that ok ? Or you prefer i close this one and start another ?